README.md

```diff
@@ -1,2 +1,112 @@
+
+The current stage of the software is *Alpha*.
+
+## Features
+- Model the dynamics of infectious diseases
+- Parameter fitting
+- Calculation of confidence intervals
+- CLI for interfacing with non Python environments (Bash, Node.JS, Matlab, etc).
+
+So far, Open-SIR provides an implementation of the SIR model and the recently new SIR based model SIR-X, developed by the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions).
+
+## Development Setup
+
+Open-SIR uses [Pipenv](https://pipenv.pypa.io/en/latest/) to automatically create a virtual environment and manage python packages. The python packages required by Open-SIR are listed in the [Pipfile](Pipfile).
+
+
+### Dependencies
+* Python 3.7
+* Pipenv
+
+### Installation
+
+After cloning the repository, change the current directory to the repository via `cd open-sir` and automatically install the environment from the Pipfile using Pipenv:
+
+```
+pipenv install
+```
+
+Next, activate the Pipenv shell:
+```
+pipenv shell
+```
+
+You can run the following command to check that the installation succeeded.
+
+```
+pipenv run python open-sir.py -p '[0.95,0.38]' -i '[341555,445,0]' -t 6 > data.csv
+```
+
+## Running the tests
+
+Test the package running the tests out of the open-sir Pipenv environment.
+```
+pipenv run test
+```
+
+## Usage example
+
+You can use Open-SIR to create a 6 days prediction of the number of susceptible (S), infected (I) and removed (R) population. 
+The initial conditions '-i' represent Ealing data as of 04/04/2020. The parameters '-p' provide a prediction in the hypothetical 
+case that no lockdown would be taking place.
+
+### Command line interface
+In the Pipenv shell, check that the installation was successful calling the CLI open-sir.py
+
+```
+python open-sir.py -p '[0.95,0.38]' -i '[341555,445,0]' -t 6 > data.csv
+```
+
+The output of open-sir.py is a .csv file with the predictions.
+
+### API
+
+You can replicate the predictions of the CLI with the following python script:
+
+```
+from models import SIR
+my_SIR = SIR() # Initialize an empty SIR model
+params = [0.95,0.38] # Define model parameters
+w0 = [341555,445,0] # Define initial conditions
+my_SIR.set_params(p=params,initial_conds=w0) # Set model parameters
+n_days = 6 # Define the amount of days to predict
+my_SIR.solve(n_days, n_days+1) # Call model.solve functions
+sol = my_SIR.fetch() # Fetch model solution
+```
+
+### Try the Jupyter Notebook
+
+Open and run the Jupyter Notebook [SIR-X.ipynb](SIR-X.ipynb) to:
+* Get an overview of the SIR model
+* Explore case studies
+
+And learn how the API can be used to:
+
+* Build compartmental models
+* Fit parameters to existing data 
+* Predict susceptible, infected and removed population
+* Calculate confidence intervals of the predictions
+
+
+### Coding style
```

Comment:
Coding Style shouldn't be at the same level of "Usage"

Reply:
Yes, and also it was floating in a wrong subsection. I moved it following [PurpleBooth Template](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) below the general tests on the dev setup. Whenever we have the package, we can re-sort the README but that should be part of other PR

See f7dcb95